test/integration/controller/jobset_controller_test.go

```diff
@@ -1326,76 +1318,16 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 				},
 			},
 		}),
-		ginkgo.Entry("startupPolicy with InOrder; suspend should keep jobs suspended", &testCase{
```

Comment:
I'm really concerned with deleting tests.

This seems like we are losing coverage.

Reply:
The thing is, the following test testing resuming a jobset basically start with what this test would test, so I didn't have a feeling we need to keep these, but I can surely put them back.

Reply:
I see what you mean now. I am fine with this actually.